experimenter/experimenter/nimbus-ui/src/components/Summary/TableQA/QAEditor.tsx

```diff
@@ -0,0 +1,193 @@
+      label: NimbusExperimentQAStatusEnum.YELLOW,
+      value: NimbusExperimentQAStatusEnum.YELLOW,
+      description: "🟡 YELLOW ",
+    },
+    {
+      label: NimbusExperimentQAStatusEnum.GREEN,
+      value: NimbusExperimentQAStatusEnum.GREEN,
+      description: "🟢 GREEN ",
+    },
+  ];
+
+  const onClickCancel = useCallback(
+    () => setShowEditor(false),
+    [setShowEditor],
+  );
+
+  const [qaStatusField, setQaStatus] =
+    useState<NimbusExperimentQAStatusEnum | null>(qaStatus);
+
+  const handleSave = handleSubmit(async (data: DefaultValues) => {
+    if (isLoading) return;
+    data.qaStatus = qaStatusField;
+    await onSubmit(data);
+  });
+
+  return (
+    <section id="qa-editor" className="mx-4" data-testid="QAEditor">
+      <h3 className="h4 mb-3 mt-4">
+        <Row>
+          <Col>QA</Col>
+          <Col className="text-right">
+            <Button
+              data-testid="qa-edit-save"
+              onClick={handleSave}
+              disabled={isLoading}
+              variant="primary"
+              size="sm"
+              className="mx-1"
+            >
+              Save
+            </Button>
+            <Button
+              data-testid="qa-edit-cancel"
+              onClick={onClickCancel}
+              disabled={isLoading}
+              variant="secondary"
+              size="sm"
+              className="mx-1"
+            >
+              Cancel
+            </Button>
+          </Col>
+        </Row>
+      </h3>
+      <FormProvider {...formMethods}>
+        <Form
+          noValidate
+          onSubmit={handleSave}
+          validated={isSubmitted && isValid}
+          data-testid="FormQA"
+        >
+          {submitErrors["*"] && (
+            <Alert data-testid="submit-error" variant="warning">
+              {submitErrors["*"]}
+            </Alert>
+          )}
+
+          <Form.Group as={Row} controlId="qaStatus" className="mb-0 pl-1">
+            <Form.Group data-testid="qa-status">
+              <Row className="ml-0 flex-fill pl-0">
+                <Col className="ml-2 mr-0 pr-0 w-25">
+                  <Form.Label>
+                    <p className="font-weight-bold">QA Status: </p>
+                  </Form.Label>
+                </Col>
+                <Col className="ml-4 flex-fill pl-0 pr-8 mr-4">
+                  <Form.Control
+                    as="select"
+                    className="ml-0 mr-0"
+                    value={qaStatusField as NimbusExperimentQAStatusEnum}
+                    onSubmit={handleSave}
+                    onChange={(e) =>
+                      setQaStatus(
+                        e.target
+                          ? (e.target.value as NimbusExperimentQAStatusEnum)
+                          : null,
+                      )
+                    }
+                    id="qaStatus"
+                    {...{ "data-testid": "qaStatus" }}
+                  >
+                    <QaStatusOptions options={qaStatusOptions} />
```

Comment:
qaStatusOptions is a constant above. this doesn't need to be passed in.

Additionally, you can just expand this inline.
```suggestion
                    {qaStatusOptions.map(
                      (item, idx) => <option key={idx} value={item.value}>{item.description}</option>
                    )}
```

Reply:
That's what `QAStatusOptions` was doing. I'll just inline that instead of using `QAStatusOptions` 😄